client/src/components/ReportMap.js

```diff
@@ -2,15 +2,17 @@ import API from "api"
 import { gql } from "apollo-boost"
 import Leaflet from "components/Leaflet"
 import {
-  PageDispatchersPropType,
   mapPageDispatchersToProps,
+  PageDispatchersPropType,
   useBoilerplate
 } from "components/Page"
```

Comment:
Please keep the order of the imports from a package alphabetically sorted (case-sensitive).

Reply:
Hm, interestingly VS Code now also sorts them case-insensitive for me (it used to put the caps ones first). I can live with that. :wink: